### PR TITLE
Refinement of HDP Sandbox to Databricks QS and linked documents

### PR DESCRIPTION
**Pull request by benradstone**
Touched on a few documents based on feedback.

### CODE REVIEW

File: docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Log in to your VM prior to starting these steps.
 
 4. Enter `y` when asked whether to use the HDP sandbox.
 
-5. You have now completed the setup, run the following to start your containers:
+5. You have now completed the setup, run the following to create and start your containers:
```
Comment (catrionabarker) [inline docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md]: Even shorter would be
You have now completed the setup, to create and start your containers run:

File: docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ Log in to your VM prior to starting these steps.
...
 
    `docker-compose up -d`
```
Comment (catrionabarker) [inline docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md]: I don't think 'please wait until this is done.' is necessary here. You've stated what will happen and so people know what to look/wait for

Comment (benradstone) [reply docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md]: Have you commented on the wrong line here? It's referencing the same line as earlier.

Comment (benradstone) [reply docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md]: Oh I see, it's the line directly below.

Comment (benradstone) [reply docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md]: What I foresee happening is users may attempt to proceed in the QS, and will not be able to log into Ambari in the next step (and think something is wrong).

Comment (benradstone) [reply docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md]: Test

File: docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md [hunk 3]
```diff
@@ -137,21 +137,29 @@ Follow the steps below to demonstrate live replication of HCFS data and Hive met
 
    Both rules should be displayed afterwards.
```
Comment (catrionabarker) [inline docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md]: replace should with will maybe?

Comment (catrionabarker) [reply docs/quickstarts/installation/hdp_sandbox_lhv_client-adlsg2_lan.md]: Or even 'Both rules will now be displayed'

**Comment by benradstone** [comment]
Changes made, please re-review.

**Pull request merged**